### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ShahierNashaat**
Pull request for review styling for bookstore project using css3 and react.

Implement styling.

Fix linter error.

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -15,8 +15,15 @@ const BookItem = (props) => {
...
       <h2>{book.title}</h2>
-      <button type="button" onClick={removeBookfromStore}>Delete</button>
+      <div className="buttons">
+        <button type="button">Comments</button>
+        |
+        <button type="button" onClick={removeBookfromStore}>Remove</button>
+        |
+        <button type="button">Edit</button>
+      </div>
     </li>
```
Comment (DeliceLydia) [inline src/components/BookItem.js]: - [ ] There some missing part which is in the design check on [Zeplin]()
zp

There are some features that are not yet implemented that you should add them as a placeholder

File: src/App.js [hunk 2]
```diff
@@ -6,18 +6,25 @@ import Categories from './components/Categories';
...
           <ul>
             <li>
-              <Link to="/">Books</Link>
+              <Link to="/" className="active" onClick={addActiveClass}>BOOKS</Link>
             </li>
-            |
             <li>
-              <Link to="/categories">Categories</Link>
+              <Link to="/categories" onClick={addActiveClass}>CATEGORIES</Link>
             </li>
```
Comment (DeliceLydia) [inline src/App.js]: - [ ] In the navbar there is an icon missing
nav

Comment (DeliceLydia) [reply src/App.js]: Check the design to see what I am talking about, make sure your page looks like exactly the design

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Status: Change Requested

## Hi @ShahierNashaat, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -6,20 +6,34 @@ import Categories from './components/Categories';
...
+      document.querySelector('nav ul li a').classList.add('active');
+    } else {
+      e.target.classList.add('active');
+    }
+  };
+
   return (
     <Router>
       <div>
         <nav>
```
Comment (TedLivist) [inline src/App.js:24]: - [ ] **[OPTIONAL]** Currently, your Navbar is missing the background color in the template.

  I would suggest adding a className to this `<nav>` and then in your **App.css** give it a `background-color` property with the appropriate color (like **#fff**).

  shah-nav

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ShahierNashaat,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**